src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c

```diff
@@ -1,10 +1,29 @@
 /*
  * File: z_shot_sun.c
  * Overlay: ovl_Shot_Sun
- * Description: Song of Storms Fairy
+ * Description: Song of Storms Fairy spawner and Lake Hylia Sun hitbox
  */
 
 #include "z_shot_sun.h"
+#include <vt.h>
+
+// TODO: Remove when `EnItem00` gets moved to a header file
+struct EnItem00;
```

Comment:
we talked a bit in discord about this and think it would be best if the header for enitem00 is moved to z64actor.h. are you ok with doing it in this pr?
Would also move the other header for the actor in en_a_keep.c

Reply:
Done. How does it look?

Reply:
looks great thanks